scss/mixins/_buttons.scss

```diff
@@ -42,16 +42,16 @@
   }
 }
 
-@mixin button-outline-variant($color, $color-hover: #fff) {
+@mixin button-outline-variant($color, $color-hover: #fff, $active-background: darken($color, 7.5%), $active-border: darken($color, 10%)) {
```

Comment:
Since you are using the same parameters as in `button-variant` wouldn't it be better is we put outline variant under the same umbrella as normal variants? 
That way we keep them DRY and give less places to change to users.
What do you think?